src/lib/formats/tzx_cas.cpp

```diff
@@ -228,11 +233,12 @@ static int pause_one_millisec( int16_t **buffer )
 
 static int tzx_cas_handle_block( int16_t **buffer, const uint8_t *bytes, int pause, int data_size, int pilot, int pilot_length, int sync1, int sync2, int bit0, int bit1, int bits_in_last_byte )
 {
-	int pilot_samples = tcycles_to_samplecount(pilot);
-	int sync1_samples = tcycles_to_samplecount(sync1);
-	int sync2_samples = tcycles_to_samplecount(sync2);
-	int bit0_samples = tcycles_to_samplecount(bit0);
-	int bit1_samples = tcycles_to_samplecount(bit1);
+	float t_scale=1;
+	int pilot_samples = tcycles_to_samplecount(pilot,t_scale);
+	int sync1_samples = tcycles_to_samplecount(sync1,t_scale);
+	int sync2_samples = tcycles_to_samplecount(sync2,t_scale);
+	int bit0_samples = tcycles_to_samplecount(bit0, t_scale);
+	int bit1_samples = tcycles_to_samplecount(bit1,t_scale);
```

Comment:
Please don’t code like your space bar is broken.

Reply:
Fixed